API-strategie-modulen/API-strategie-mod-geo/request-response.md

```diff
@@ -1,7 +1,10 @@
 # Request and response
-There are several challenges related to handling geospatial features in REST APIs. There is a distinction between retrieving geometries in the result (response) and supplying a geometry in the call (request). 
 
-Note that when requesting information, for example about cadastral parcels, users do not necessarily require the geometry. A name or parcel ID may be sufficient. Geometries can also be part of the request, for example as a spatial filter or when creating or updating a spatial feature. When geometry is part of the request or response, several rules apply, which are described below. 
+Requesting resources and getting results back is an important part of any API. This is no different for REST APIs that handle geospatial data. There are, however, some specific aspects when dealing with geospatial data in REST APIs. The most important aspects are described in this chapter: 
+- how to supply a geometry in the call (request)
```

Comment:
Wat wordt hier bedoeld met supply a geometry?
1. Het aanleveren van geometrie als onderdeel van een resource in een request body van een POST request, bv. als een resource moet worden opgeslagen.
2. Het filteren van resources d.m.v. een geometrie zoals een bbox?
Ik denk dat daar een onderscheid in zit en dat dit punt misschien beter in twee afzonderlijke punten gesplitst kan worden. 
Voor het aanleveren van geometrie als onderdeel van een resource wordt de Ontent-Crs header gebruikt.
Voor het filteren van resources die bv. (deels) binnen een zoek gebied (geometrie) vallen worden bv. filter-crs en bbox-crs parameter gebruikt. 
Voor het volgende punt (encoding van geometrie in de response) wordt de crs parameter gebruikt en in de response de Content-Crs header.

Reply:
Ik heb hier nog wat voor aangepast. 'supply a geometry' was al herschreven als 'supply a spatial filter'. Ik heb nog een Note toegevoegd over aanleveren van geometrie als onderdeel van een resource. 

Wil je nog een keer reviewen @strijm ?